config/schedule.rb

```diff
@@ -1 +1,13 @@
-# This file is overwritten on deploy
+set :output, error: 'log/cron.error.log', standard: 'log/cron.log'
+
+schedule_task_prefix = ENV.fetch('SCHEDULE_TASK_PREFIX', '/usr/local/bin/govuk_setenv publisher')
```

Comment:
Is there a reason for looking for this in an env var?  It's not likely to change particularly, is it?

Reply:
That's to make it so that it's possible to set this up to run outside of our infrastructure. As it stands your only way to run this is to have `/usr/local/bin/govuk_setenv` available. Which can be quite a pain point, but doesn't actually solve any current problems.